content/docs/install/completion.md

```diff
@@ -16,11 +17,11 @@ For example:
 
 ```dvc
 $ dvc r # Press [tab] key
- -- dvc commands --
-remote  -- Manage remote storage configuration.
-remove  -- Remove outputs of DVC-file.
-repro   -- Check for changes and reproduce DVC-file and dependencies.
-root    -- Relative path to project's directory.
+Completing dvc commands
+remote  -- Set up and manage data remotes.
+remove  -- Remove stage entry and unprotect outputs
+repro   -- Reproduce complete or partial pipelines by executing their stages.
```

Comment:
Minor: this goes over 72 chars and produces a horizontal scroll bar:

![image](https://user-images.githubusercontent.com/1477535/85360701-77de8580-b4df-11ea-83ad-110c07c99d67.png)

Reply:
I'd just remove the `.` longest line.

Reply:
And maybe add `...` after the last line (under `run`)

Reply:
Ah no never mind those are all the `r` commands.

Reply:
Or maybe use another letter as example.